src/sessions/session-manager.ts

```diff
@@ -166,6 +166,19 @@ export class SessionManager {
     return Promise.all(promises);
   }
 
+  updateAudioVolume (volume: number): void {
+    const sessions = this.getAllActiveSessions();
+    this.log('info', 'Updating volume for all active sessions', {
```

Comment:
Is there a use case where this could be spammed and creates a ton of log messages? For example, could this get triggered by a hardware/keyboard control, or slider that would trigger this every time a slider is incremented?

Reply:
I think we should be ok here. it won't be triggered by system volume, and sliders should only update "onBlur" or equivalent.